velox/docs/functions/presto/window.rst

```diff
@@ -132,17 +132,19 @@ returned. It is an error for the offset to be zero or negative.
 
 .. function:: lag(x[, offset [, default_value]]) -> [same as input]
 
-Returns the value at ``offset`` rows before the current row in the partition. If
-there is no such row, the ``default_value`` is returned, or if it is not
-specified ``null`` is returned. Offsets start at ``0``, which is the current
-row. The default ``offset`` is ``1``.
+Returns the value at ``offset`` rows before the current row in the window partition.
+Offsets start at ``0``, which is the current row. The default ``offset`` is ``1``.
+The offset can be any scalar expression. If the offset is ``null``, ``null`` is
+returned. If the offset refers to a row that is not within the partition, the
+``default_value`` is returned, or if it is not specified ``null`` is returned.
```

Comment:
> if it is not specified ``null`` is returned.

What does "it" refer to here? Is it "offset"? Is so, what does it mean for offset to not be specified? You said above that default value for offset is 1. Please, clarify.